src/docs/docsite.json

```diff
@@ -159,6 +159,7 @@
         { "page": "options_reference" },
 
         { "heading": "Release Notes"},
+        { "page": "notes-1.5.x" },
```

Comment:
This needs an entry above to define the "notes-1.5.x" key, around here: https://github.com/pantsbuild/pants/pull/5536/files#diff-662ec19c17add6c9e6b532bbd7a0ba57R56

Reply:
Added. Thanks John